<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainCSCUSet</class>
 <widget class="QWidget" name="MaintainCSCUSet">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>767</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: \*url();
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="rightMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,11">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1,1">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout" columnstretch="2,3">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <property name="verticalSpacing">
            <number>9</number>
           </property>
           <item row="6" column="0">
            <widget class="QLabel" name="labelServer2Address">
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>服务器2地址：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="labelServer1PortNumber">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>服务器1端口号：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelIP">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>本地IP地址：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="labelDCTerminalCount">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>直流终端数量：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="LineEditKeyboard" name="lineEditSingleNum">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="labelACTerminalCount">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>交流终端数量：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="LineEditKeyboard" name="lineEditDCTerminalCount">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="LineEditKeyboard" name="lineEditDefaultGateway">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="LineEditKeyboard" name="lineEditServer1PortNumber">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="LineEditKeyboard" name="lineEditDNSServer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="LineEditKeyboard" name="lineEditIP">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="LineEditKeyboard" name="lineEditServer1Address">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="LineEditKeyboard" name="lineEditStationAddress">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="labelZigbeeAddress">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Zigbee地址：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="labelThreePhaseTerminalCount">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>三相终端数量：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelDNSServer">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>DNS服务器：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelServer1Address">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>服务器1地址：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="LineEditKeyboard" name="lineEditZigbeeAddress">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelDefaultGateway">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>默认网关：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="labelCSCUPortAddress">
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>集控器站地址：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="LineEditKeyboard" name="lineEditThreePhaseTerminalCount">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="labelServer2PortNumber">
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>服务器2端口号：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="labelServer3Address">
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>服务器3地址：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="labelServer3PortNumber">
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>服务器3端口号：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="LineEditKeyboard" name="lineEditServer2Address">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="LineEditKeyboard" name="lineEditServer2PortNumber">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="LineEditKeyboard" name="lineEditServer3Address">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="LineEditKeyboard" name="lineEditServer3PortNumber">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>318</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSave">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 18pt ; } </string>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditKeyboard</class>
   <extends>QLineEdit</extends>
   <header>LineEditKeyboard.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
